api.web.yaml

```diff
@@ -18,6 +18,13 @@ paths:
                 $ref: "#/components/schemas/ProfileResponse"
         default:
           description: Unexpected error
+    post:
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ProfileResponse"
```

Comment:
I think this should be a new schema e.g. `ProfileUpdate`, then will also need to define the responses below this e.g. https://github.com/ucl-arc-tre/portal/blob/9024182498ed647cbb578499a5154809be65a14d/api.web.yaml#L14-L20